### DISCUSSION TITLE
Can I generate INT signal with LIS2DW12 library?

### DISCUSSION THREAD

**Discussion by myst729** [stm32duino/Arduino_Core_STM32#2355] [Libraries]
My plan is to detect device movement with LIS2DW12. And use the INT signal as the wakeup source to STM32F401 (HIGH on pin PA0). Is it possible, any ideas?

**Comment by fpistm** [comment]
Maybe @cparata has some input for this use case?

**Comment by cparata** [comment]
Hello @myst729 ,
you can give a look at [this sketch](https://github.com/stm32duino/X-NUCLEO-IKS01A3/blob/main/examples/X_NUCLEO_IKS01A3_LIS2DW12_WakeUpDetection/X_NUCLEO_IKS01A3_LIS2DW12_WakeUpDetection.ino) that shows how to use the wake up feature of the LIS2DW12 with the interrupt.
Best Regards,
Carlo

**Reply by fpistm** [reply]
Thanks @cparata for the quick answer.
